rust/Cargo.lock

```diff
@@ -0,0 +1,1320 @@
+[[package]]
+name = "adler32"
+version = "1.0.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+
+[[package]]
+name = "arrayvec"
+version = "0.4.7"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "nodrop 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
+[[package]]
+name = "atom_syndication"
```

Comment:
the lock file we can take or leave, in the case of a binary output it provides determinability. If this is going to be a cap library, then we'd leave it off.

rust ecosystem adheres to versions much more strictly than go. It's very rare to find direct dependencies on commits instead of some tagged version.